en/developer_guide/custom_content_attribute_display.rst

```diff
@@ -0,0 +1,50 @@
+Customize Content Attribute Display
+===================================
+
+Context
+-------
+
+In order to have minimum constraints on contents, Open Orchestra allows you to store any type
+of ``value`` in the ``ContentAttribute``.
+
+Sometimes, there is no easy way to render the value to the backoffice user.
+A custom ``ContentAttribute`` could an array, which cannot be displayed directly in javascript.
+In that way the ``ContentAttribute`` class provide a specific attribute ``stringValue`` which is the
+representation of the ``value`` in an HTML string.
+
+Transformers
+------------
+
+Open Orchestra uses a ``ValueTransformer`` strategy manager to generate the HTML strings for
+multiple ``value`` type. Open Orchestra already provides some strategies located in
+``OpenOrchestra\Backoffice\ValueTransformer\Strategies``.
+
+Strategy creation
+~~~~~~~~~~~~~~~~~
+
+Service used as ``ValueTransformer`` must be tagged as ``open_orchestra_backoffice.value_transformer.strategy``.
+
+.. code-block:: yaml
+
+    tags:
+        - { name: open_orchestra_backoffice.value_transformer.strategy }
+
+Such services also need to implement ``OpenOrchestra\Backoffice\ValueTransformer\ValueTransformerInterface``.
+
+Strategy override
+~~~~~~~~~~~~~~~~~
+
+Strategies can be overriden by creating a new service which return the same value for the ``getName`` method.
+
+Strategy access
+~~~~~~~~~~~~~~~
+
+Open Orchestra uses the ``OpenOrchestra\Backoffice\ValueTransformer\ValueTransformerManager``
+as an easy way to access every transformation strategies.
+All ``ValueTransformer`` strategies are registered in the ``ValueTransformerManager``.
+The ``transform`` method of the manager:
+- searches the first transformation strategy supporting the current ``fieldType`` and ``value``
+of the ``ContentAttribute``
+- uses this strategy's ``transform`` method to return the ``stringValue``
+
+If no transformer supports the actual parameters, the method returns the original ``value``.
```

Comment:
Some examples are required here, taking a complex array and showing it's representation could be helpfull for the reader. It could also be used to describe the manager/transformer phase.
Maybe that exemple could be mentionned earlier in the intriduction and solved here.